cache/redis_client.go

```diff
@@ -89,8 +91,8 @@ type RedisClientConfig struct {
 	// If set to 0, concurrency is unlimited.
 	MaxGetMultiConcurrency int `yaml:"max_get_multi_concurrency" category:"advanced"`
 
-	// GetMultiBatchSize specifies the maximum size per batch for mget.
-	GetMultiBatchSize int `yaml:"get_multi_batch_size" category:"advanced"`
+	// MaxGetMultiBatchSize specifies the maximum size per batch for mget.
+	MaxGetMultiBatchSize int `yaml:"max_get_multi_batch_size" category:"advanced"`
```

Comment:
This change is fine (actually wanted to keep consistency with the memcached client) but please mention it in the CHANGELOG.

Reply:
The first time this is added, the config name and variable name missing max prefix. Ok I will add this new config name in the dskit changelog.

Reply:
Also, the renaming above is to make the config name consistent with Memcached.